Dockerfile

```
@@ -1,11 +1,13 @@
 FROM python:3.5-slim-stretch
-RUN apt-get update > /dev/null && apt-get install -y --reinstall build-essential && \
-    apt-get install -y --no-install-recommends \
-    curl \
-    file \
-    git \
-    default-libmysqlclient-dev \
-    mecab \
-    mecab-ipadic-utf8 \
-    libmecab-dev \
-    swig
\ No newline at end of file
+RUN apt-get update > /dev/null && apt-get install -y --no-install-recommends \
+    build-essential=12.3 \
+    curl=7.52.1-5+deb9u11 \
+    file=1:5.30-1+deb9u3 \
+    git=1:2.11.0-3+deb9u7 \
+    default-libmysqlclient-dev=1.0.2 \
+    mecab=0.996-3.1 \
+    mecab-ipadic-utf8=2.7.0-20070801+main-1 \
+    libmecab-dev=0.996-3.1 \
+    swig=3.0.10-1.1 \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
```

What about installing corresponding python-dependency as well? Specifying it here should result in `Requirement already satisfied: mecab-python3==0.996.1 in ...` in `user-app-server-side`-build which should make it faster.
```suggestion
    && rm -rf /var/lib/apt/lists/*
    && rm -rf /var/lib/apt/lists/*
RUN pip install mecab-python3==0.996.1
```

---

@Luke31 Thanks for your comment! Nice! I'll add it!

---

I checked the build time but it takes only 7.8s sec, so I think it's ok not to include it in this repo?

```
○ DOCKER_BUILDKIT=1 docker build -t python-mecab-builder .
[+] Building 65.1s (7/7) FINISHED
 => [internal] load .dockerignore                                                                                                                                           0.0s
 => => transferring context: 2B                                                                                                                                             0.0s
 => [internal] load build definition from Dockerfile                                                                                                                        0.1s
 => => transferring dockerfile: 524B                                                                                                                                        0.0s
 => [internal] load metadata for docker.io/library/python:3.5-slim-stretch                                                                                                  3.3s
 => [1/3] FROM docker.io/library/python:3.5-slim-stretch@sha256:6c621c4cc660b710f8704644eaf48fda20ee2becff0eddfa7e972720136a0eb1                                            7.2s
 => => resolve docker.io/library/python:3.5-slim-stretch@sha256:6c621c4cc660b710f8704644eaf48fda20ee2becff0eddfa7e972720136a0eb1                                            0.0s
 => => sha256:6c621c4cc660b710f8704644eaf48fda20ee2becff0eddfa7e972720136a0eb1 1.22kB / 1.22kB                                                                              0.0s
 => => sha256:5fe3828b75a2d1f997fed4dff51cae4c3148b82b30e27da716b88f593e70fc68 1.37kB / 1.37kB                                                                              0.0s
 => => sha256:6417363e23d57863de11b99fa06f048982c7d0c969afdc05897b193a501f5902 8.32kB / 8.32kB                                                                              0.0s
 => => sha256:75cb2ebf3b3cb9c601a22edc7d9a120749da33999b30fba22b6cf2b90ee92823 22.52MB / 22.52MB                                                                            4.5s
 => => sha256:2fdcfea9efdf8e3e507cc1da20713b3da54f5af9094af9b9bbb8c22187fcfbbb 2.49MB / 2.49MB                                                                              1.2s
 => => sha256:ec9a68f2d858ab04cf2fc228e915c9e0cc4dcb922ab37204cdf6ba2f8309d328 9.21MB / 9.21MB                                                                              2.8s
 => => sha256:85f1325a9c5814d4afcb072b2c2404be2842cff26960b6d4590969bef8a80334 240B / 240B                                                                                  1.7s
 => => sha256:c73a0199e06cfba15580b94c983d2554aaf344c3bb1352b070d27ede7931c34f 2.40MB / 2.40MB                                                                              2.8s
 => => extracting sha256:75cb2ebf3b3cb9c601a22edc7d9a120749da33999b30fba22b6cf2b90ee92823                                                                                   1.1s
 => => extracting sha256:2fdcfea9efdf8e3e507cc1da20713b3da54f5af9094af9b9bbb8c22187fcfbbb                                                                                   0.2s
 => => extracting sha256:ec9a68f2d858ab04cf2fc228e915c9e0cc4dcb922ab37204cdf6ba2f8309d328                                                                                   0.5s
 => => extracting sha256:85f1325a9c5814d4afcb072b2c2404be2842cff26960b6d4590969bef8a80334                                                                                   0.0s
 => => extracting sha256:c73a0199e06cfba15580b94c983d2554aaf344c3bb1352b070d27ede7931c34f                                                                                   0.3s
 => [2/3] RUN apt-get update > /dev/null && apt-get install -y --no-install-recommends     build-essential=12.3     curl=7.52.1-5+deb9u11     file=1:5.30-1+deb9u3     gi  43.0s
 => [3/3] RUN pip install mecab-python3==0.996.1                                                                                                                            7.8s
 => exporting to image                                                                                                                                                      3.6s
 => => exporting layers                                                                                                                                                     3.6s
 => => writing image sha256:83c07eecb20724de20d4f5b80496c22bd015ff6bcc55511e1e14ef3260cfceb8                                                                                0.0s
 => => naming to docker.io/library/python-mecab-builder                                                                                                                     0.0s
```

---

Ok that's true, that's actually pretty fast. So we wouldn't gain very much of it.